app/controllers/carl_people_controller.rb

```diff
@@ -0,0 +1,47 @@
+class CarlPeopleController < ApplicationController
+  #----------------- SHOW A PERSON --------------------
+  def show
+    @person = Person.find(params[:id])
+    @tasks_count = @person.tasks.count
+    @tasks_wip = @person.tasks.where(date_complete: nil).count
+    @tasks_done = @tasks_count - @tasks_wip
+  rescue
```

Comment:
you *should* be able to just skip the rescue entirely. I think it'll still raise an exception in development, but show the 404 page in production, but I'm not 100% about what it does in development, you should double check.